model/src/main/scala/uk/gov/ons/addressIndex/model/config/AddressIndexConfig.scala

```diff
@@ -5,7 +5,11 @@ case class ElasticSearchConfig(
   cluster: String,
   uri: String,
   indexes: IndexesConfig,
-  shield: ShieldConfig
+  shield: ShieldConfig,
+  defaultLimit: Int,
+  defaultOffset: Int,
+  maximumLimit: Int,
```

Comment:
_"It would be good to see the request only constructing the pagination (e4s) req if the option is some. It seems to be forced via defaults"._

Looking at implementing this one I see another issue. As I have coded it, the API's defaults are used if no parameters are specified. If no parameters are sent through to ES then its defaults will apply. I think it is preferable for the API to have control over the defaults.